fattmerchant/Merchant.py

```diff
@@ -281,3 +282,336 @@ def set_merchant_id(self, id):
+            "contact_name": "Fattmerchant",
+            "contact_email": "info2@fattmerchant.com",
+            "contact_phone": "8555503288",
+            "address_1": "25 Wall Street",
+            "address_2": "Suite 1",
+            "address_city": "Orlando",
+            "address_state": "FL",
+            "address_zip": "32801"
+            }
+
+        .. note::
+            Authentication Token and Team Admin Status Required
+
+        """
+        pass
+
+    def team_edit_branding_info(self, branding_info:dict):
+        """somethign
+        
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        This resource gives merchant admins the ability to change their company's branding.
+        branding may be uploaded as either a PNG or JPEG image file.
+        A merchant's branding will appear anywhere a merchant's object is returned.
+        The merchant branding will be securely hosted on Amazon S3 bucket and recorded in our database.
+
+        Arguments:
+            branding_info {dict} -- [description]
+
+        example
+
+        .. code-block:: JSON
+
+            {
+                "name": "Mercedes-Benz",
+                "image": "logo.png"
+            }
+
+        """
+        pass
+    
+    def team_list_members(self):
+        """
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        Retrieves all the users on a merchants's team, showing team-oriented details and team_role.
+        Used for finding all members of a team and listing them by roles.
+        """
+        pass
+    
+    def team_create_member(self,user_info):
+        """
+        .. note::
+            Authentication Token and Team Admin Status Required
+        
+        This creates a whole new user account in your merchant team.
+        Automatically adds the user to the merchant team.
+        Can only be used by users with team_rule of "admin".
+        This is one of two ways to create a new user account; the other being POST /self.
+        Requires a valid email, password and name at minimum.
+        If team_role is not selected, it will default to "full".
+        Used by team admins who want to add members to their merchant account who do not already have a Fattmerchant account.
+       
+        
+        Arguments:
+            user_info {[type]} -- user information in a dictionary
+        
+        example 
+
+        .. code-block:: JSON
+
+            {
+            "email": "daniel+1024@fattmerchant.com",
+            "password": "bottomline",
+            "password_confirmation": "bottomline",
+            "name": "Bob Dylan",
+            "team_role": "full",
+            "team_enabled": true,
+            "send_verification_email": true,
+            "url": "https://omni.fattmerchant.com/#/verify/"
+            }
+        """
+
+        pass
+    
+    def team_create_api_key_user(self, name:str, role:str): 
+        """creates api key 
```

Comment:
1. Pull the method description up to here

2. Place the note below the description.